css/properties/inline-size.json

```diff
@@ -196,8 +196,7 @@
                 "version_added": false
               },
               "firefox": {
-                "prefix": "-moz-",
-                "version_added": "3"
+                "version_added": "38"
```

Comment:
I don't understand why this would be 38. Can you explain?

Reply:
Both of your comments are related to one another.  All of the css.[min/max][height/block-size/inline-size] features were introduced to the W3 spec at the same time, and based upon the data I've found from CanIUse and Firefox's commits (which I totally should have saved for this exact reason), it didn't seem like any of these properties were implemented before any of their siblings.

Reply:
I'm reverting and tweaking these changes in favor of working alongside #3624.